src/node_character.h

```diff
@@ -2,6 +2,7 @@
 #include "mesh.h"
 #include "node.h"
 #include "node_model.h"
+#include "rendering_mode.h"
```

Comment:
```suggestion
#include "modules/rendering_mode.h"
```
wont build for me unless I import this correctly